sdk/src/main/java/com/meniga/sdk/models/sync/operators/MenigaSyncOperations.java

```diff
@@ -9,7 +9,7 @@
  */
 public interface MenigaSyncOperations {
 
-	Result<MenigaSync> startSync(Long realmUserId, long timeout);
+	Result<MenigaSync> startSync(Long realmUserId, String sessionToken, long timeout);
```

Comment:
You can add `@Nullable` to the new parameter.

Reply:
Isn't String inherently nullable?

Reply:
Sure

Reply:
The signature (+ annotations) can be the same as in `MenigaSyncOperationsImp`

Reply:
Fixed